ROADMAP.md

```diff
@@ -0,0 +1,31 @@
+---
+title: TiKV Roadmap
+category: Roadmap
+---
+
+# TiKV Roadmap
+
+This document defines the roadmap for TiKV development.
+
+- [ ] Raft
+    - [x] Region merge
+    - [ ] Local read thread
+    - [ ] Multi-thread raftstore
+    - [x] None voter
+    - [x] Pre-vote
+    - [ ] Multi-thread apply pool
+    - [ ] Split region in batch
+    - [ ] Raft Engine
+- [x] RocksDB 
+    - [x] DeleteRange
+    - [ ] BlobDB 
+- [x] Transaction
+    - [x] Optimize transaction conflicts
+    - [ ] Distributed GC
+- [x] Coprocessor
+    - [x] Streaming
+- [ ] Tool
```

Comment:
Can you please add the TiKV client (Rust crate) here? :)

Reply:
@siddontang can we?

Reply:
yes